Branch-SDK/Branch-SDK/BranchUniversalObject.m

```diff
@@ -132,6 +132,16 @@ - (void)showShareSheetWithLinkProperties:(BranchLinkProperties *)linkProperties
     }
     UIActivityViewController *shareViewController = [[UIActivityViewController alloc] initWithActivityItems:items applicationActivities:nil];
     
+    if ([shareViewController respondsToSelector:@selector(completionWithItemsHandler)]) {
+        shareViewController.completionWithItemsHandler = ^(NSString *activityType, BOOL completed, NSArray *returnedItems, NSError *activityError) {
```

Comment:
Is there a reason why the callback passed (https://github.com/BranchMetrics/iOS-Deferred-Deep-Linking-SDK/pull/270/files#diff-fdcd6f070a770d8a13ad99f764c983e0R14) in doesn't match the callback here? Wouldn't it be cleaner if you could simply have `shareViewController.completionWithItemsHandler = completion`? Just a suggestion